apps/condo/domains/notification/gql.js

```diff
@@ -23,10 +23,28 @@ const RESEND_MESSAGE = gql`
         result: resendMessage(data: $data) { status id }
     }
 `
+
+const DEVICE_REQUIRED_FIELDS = 'deviceId pushToken pushTransport meta owner { id }'
+const DEVICE_FIELDS = `{id ${DEVICE_REQUIRED_FIELDS}}`
```

Comment:
What about COMMON_FIELDS? It’s better to add it to check

Reply:
I didn't use any of COMMON_FIELDS in service, that's why I didn't pass them.

Reply:
fixed